ruby/examples/app/controllers/user_registration_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UserRegistrationController < ApplicationController
+  def new
+    @user = UserRegistration.new
+  end
+
+  def create
+    request = RegisterUser::Request.new(*user_params)
+    response = RegisterUser.new.register(request)
```

Comment:
Replied to that argument separately.

I think this is a bigger issue with: reducing number of gems, avoiding boilerplate that makes rubyists baulk, adding indirection to avoid problems that are unlikely. I'm sure these problems are bigger in other languages, just don't think we need to worry.